index.html

```diff
@@ -0,0 +1,69 @@
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="index.css">
+  </head>
+  <body>
+    <form action="contactForm.php" method="POST" class="contact-form">
+
+  <div class="cf-item">
+    <label for="fname">First Name</label>
+    <input class="cf-input" type="text" id="fname" name="firstname" placeholder="Your name..">
```

Comment:
Why the double dots?
I think these placeholders are redundant. They're not adding any extra information. Also if the next one says 'Your last name', this should probably say 'Your first name'.